src/IRPrinter.cpp

```diff
@@ -425,21 +428,31 @@ void IRPrinter::visit(const Call *op) {
     }
 
     stream << op->name << "(";
-    for (size_t i = 0; i < op->args.size(); i++) {
-        print(op->args[i]);
-        if (i < op->args.size() - 1) {
-            stream << ", ";
-        }
-    }
+    indent += 2;
+     for (size_t i = 0; i < op->args.size(); i++) {
+        stream << "\n";
+        do_indent();
+         print(op->args[i]);
+         if (i < op->args.size() - 1) {
+             stream << ", ";
+         }
+     }
+    indent -= 2;
+    stream << "\n";
+    do_indent();
     stream << ")";
 }
 
 void IRPrinter::visit(const Let *op) {
     stream << "(let " << op->name << " = ";
     print(op->value);
-    stream << " in ";
+    stream << " in";
```

Comment:
I think this change is going to make some IR that I've seen really difficult to read. It's not great the way it is now (massive blob of text with let exprs) but I think this was not good either when I tried it.